rmgpy/tools/canteramodel.py

```diff
@@ -394,6 +448,12 @@ def simulate(self):
                 newkey = get_species_identifier(key)
                 new_mol_frac[newkey] = value
 
+            if self.surface:
+                new_surface_mol_frac = {}
+                for key, value in condition.surface_mol_frac.items():  # this needs to be aspecies object
```

Comment:
typo in inline comment: aspieces -> a species

Reply:
I removed this commend because it's not necessary if you use more descriptive names for variable than key, value- like you suggested in the next comment.